src/keycloak/tasks.yaml

```diff
@@ -34,9 +34,8 @@ tasks:
     actions:
       - task: config:cacert
       - cmd: |
-          export cacert=$(cat cacert.b64)
-          yq e -i '.tls[0].cacert = env(cacert)' src/istio/values/config-tenant.yaml
-          yq e -i '.tls[0].cacert = env(cacert)' src/istio/values/config-admin.yaml
+          yq eval-all --inplace 'select(fileIndex == 0) * select(fileIndex == 1)' src/istio/values/config-tenant.yaml tls_cacert.yaml
+          yq eval-all --inplace 'select(fileIndex == 0) * select(fileIndex == 1)' src/istio/values/config-admin.yaml tls_cacert.yaml
```

Comment:
Is it correct that this shouldn't be merged until an updated identity-config image/task is actually producing `tls_cacert.yaml`?

Reply:
@mjnagel correct. This task is not actively being used anywhere so we wouldnt see anything break. The identity change is captured in my registration integration test branch which is still wip.